examples/advanced/brats18/requirements.txt

```diff
@@ -1,4 +1,4 @@
-nvflare~=2.4.0rc
+nvflare~=2.5.0rc4
```

Comment:
Do we need the rc4 here? 
Or "rc" is enough?

Did you try if rc5 come, this expression will install rc5?